section_names.yml

```diff
@@ -1,12 +1,13 @@
 section_names:
+  user apis: User APIs
```

Comment:
I'm fine with a new section like this, and I guess my mind would gloss over this name, but why "User APIs"?  I don't understand the name, I'm afraid...

Reply:
Alternative section name suggestions welcome, but the idea is to convey that a user can choose between different top-level APIs for creating plots. Matplotlib itself ships with two, namely the OO and pylab interfaces, but users can also choose to use plotnine to build their plots, or start with Pandas or Xarray and use .plot(). From a user perspective, these are all valid choices that they should consider.  Other packages in this list don't have that status, i.e. they aren't alternative APIs for users to select, but instead extensions or other additional functionality, typically orthogonally to the question of the user API.

Reply:
What about "3rd party interfaces to Matplotlib"?

Reply:
Many of the packages in other sections here could be described as interfacing to mpl, but the ones in this section are specifically alternative ways to create plots, i.e. different APIs that a user can select.

Reply:
Yes, the distinction is whether they define an interface on top of Matplotlib as a whole or whether they provide an extended API for specific additional features. Maybe "3rd party APIs to Matplotlib"?

Reply:
Or maybe "Alternative APIs for Matplotlib"?  What's important is that it's an alternative API to the standard ones, even if it came branded from the Matplotlib org (which might offer its own alternative API someday!).

Reply:
I think "Alternative APIs" is fine (could even drop "for Matplotlib" since I think that is inferred.)

I still am somewhat hesitant about this. Of the four I think only `plotnine` is  _really_ a different API (and that is all that it is), while calling `pandas` and `xarray` alternative Matplotlib APIs is a pretty reductive description of either package :wink:  However, I don't think we can get these categories precisely right for all the packages...

Reply:
Ok, I'll change it to "Alternative APIs".

Sure, pandas and xarray are not themselves alternative APIs, but they definitely do *offer* alternative APIs for Matplotlib, and moreover it's their ability to create Matplotlib plots and not their data-handling features that make them suitable for this list. In any case I'm just trying to provide a better indication to a current or potential Matplotlib user why these packages could be important to them, as their current category "Plot types" is neither accurate nor helpful in letting people realize that they could choose between the OO interface, pylab, plotnine, pandas .plot(), or xarray .plot() as different APIs they could select for making the same final plot.

Note that I propose adding a couple more packages (holoviews and hvplot) to this category in a follow-on PR https://github.com/matplotlib/mpl-third-party/pull/95 , but I kept that separate so that this category could be decided independently of adding new items to it.